### ISSUE TITLE
Normalize song metadata when aggregating for browsing

### ISSUE THREAD

**Issue by derat** [derat/nup-android#21]
As described in https://github.com/derat/nup/issues/18, I'm not doing any Unicode normalization of song metadata, resulting in separate browser listings for e.g. "µ-Ziq" (starting with MICRO SIGN) and "μ-Ziq" (starting with GREEK SMALL LETTER MU).

I think I should update `getSongOrderKey` in `Song.kt` to do some sort of normalization (NFKD?), but I also need to do something tricky in `loadAggregateData` in `SongDatabase.kt`. Right now, it uses e.g. `GROUP BY LOWER(TRIM(Artist))` to try to normalize data. If I try to also do Unicode normalization there, it'll likely break database queries. SQLite supports ICU integration to provide some degree of Unicode support per [this SO page](https://stackoverflow.com/questions/611459/how-to-sort-text-in-sqlite3-with-specified-locale), but this seems to unsurprisingly be a train wreck on Android (I guess you can load [some random SQLite library](https://github.com/NanoMichael/icu_sqlite3_for_android)?).

TL;DR this sucks
